src/quo2/components/information_box.cljs

```diff
@@ -0,0 +1,106 @@
+(ns quo2.components.information-box
+  (:require [quo.theme :as theme]
+            [quo.react-native :as rn]
+            [reagent.core :as reagent]
+            [clojure.string :as string]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.button :as quo2.button]
+            [status-im.utils.handlers :refer [<sub]]
+            [quo2.components.info-message :as info-message]
+            [status-im.async-storage.core :as async-storage]))
+
+(def themes
+  {:light {:default     {:bg     colors/white
+                         :border colors/neutral-20
+                         :icon   colors/neutral-50
+                         :text   colors/black}
+           :informative {:bg     colors/primary-50-opa-5
+                         :border colors/primary-50-opa-10
+                         :icon   colors/primary-50
+                         :text   colors/black}
+           :error       {:bg     colors/danger-50-opa-5
+                         :border colors/danger-50-opa-10
+                         :icon   colors/danger-50
+                         :text   colors/danger-50}
+           :close-button colors/black}
+   :dark  {:default     {:bg     colors/neutral-90
+                         :border colors/neutral-70
+                         :icon   colors/neutral-40
+                         :text   colors/white}
+           :informative {:bg     colors/primary-50-opa-5
+                         :border colors/primary-50-opa-10
+                         :icon   colors/white
+                         :text   colors/white}
+           :error       {:bg     colors/danger-50-opa-5
+                         :border colors/danger-50-opa-10
+                         :icon   colors/danger-50
+                         :text   colors/danger-50}
+           :close-button colors/white}})
+
+(defn get-color [key]
+  (get-in themes [(theme/get-theme) key]))
+
+(defn get-color-by-type [type key]
+  (get-in themes [(theme/get-theme) type key]))
+
+(defn id-hash [id public-key global?]
+  (if global?
+    (hash id)
+    (hash (str public-key id))))
+
+(defn close-information-box [id public-key global? closed? on-close]
+  (reset! closed? true)
+  (async-storage/set-item! (id-hash id public-key global?) true)
```

Comment:
im not sure async-storage should be used  inside component

Reply:
I am not sure about alternatives, any suggestions?
I guess we can pass this call as `on-close` while creating the component, but this will be the same for all closable information boxes and having this call inside the component encapsulates its functionality.

Reply:
yup, I also don't like having async-storage in component. maybe we can create a file `quo2.core` or `quo2.components.core` where we call load states of all information box components on start and dispatch event for storing on-close. wdyt?

Reply:
i don't think we'll have lots of states, so we could just manage state locally in the component parent view